src/codegen/llvm.zig

```diff
@@ -170,6 +169,7 @@ pub fn targetTriple(allocator: Allocator, target: std.Target) ![]const u8 {
         .code16 => "code16",
         .eabi => "eabi",
         .eabihf => "eabihf",
+        .ilp32 => if (target.cpu.arch == .aarch64) "unknown" else "ilp32",
```

Comment:
I don't understand the logic here. LLVM either has this ABI or not. So this should either always map to "unknown", or always map to "ilp32", right?

Reply:
Right. I intended to do something related to RISC-V with this ABI tag, but then decided that's a problem for a future PR, and forgot to remove this.